packages/x-components/src/x-modules/empathize/components/empathize.vue

```diff
@@ -73,6 +76,23 @@
      */
     protected isOpen = false;
 
+    /**
+     * The default slot is empty to avoid opening empathize with no content.
+     *
+     * @internal
+     */
+    protected isEmptySlot: boolean | Element = true;
+
+    /**
+     * Detects if the default slot has been replaced.
+     *
+     * @internal
+     * @returns True if the slot is empty, the slot element if it has content.
+     */
+    updated(): boolean | Element {
+      return (this.isEmptySlot = !!this.$refs.isEmptySlot);
```

Comment:
Probably you've already thought about it and you know why it wouldn't work but, just in case: Couldn't we check the content of the default slot instead of adding a span and check whether is it visible or not?
```suggestion
      return (this.isEmptySlot = !this.$slots.default?.length);
```

Reply:
I think we should take advantage of `vue's` features, in this case of its `$refs` one, instead of relying in `CSS` or query selectors.

Regarding the actual code, you don't need to `return` anything here and a simple `assignment` should be enough.

```suggestion
      this.hasContent = !this.$refs.noContent;
```